templates/tfengine/components/project/healthcare_datasets/main.tf

```diff
@@ -0,0 +1,59 @@
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */ -}}
+
+{{range get . "healthcare_datasets"}}
+module "{{resourceName .name}}" {
+  source  = "terraform-google-modules/healthcare/google"
+  version = "~> 1.0.0"
+
+  name     = "{{.name}}"
+  project  = var.project_id
+  location = "{{get . "location" $.healthcare_location}}"
+
+  {{hclField . "iam_members" false}}
+
+  {{if has . "dicom_stores" -}}
```

Comment:
"dicom_stores" should be in the example, or is it mutually exclusive from fhir_stores?

Reply:
Done.